mappings/net/minecraft/client/MinecraftClient.mapping

```diff
@@ -385,7 +387,12 @@ CLASS net/minecraft/class_310 net/minecraft/client/MinecraftClient
 	METHOD method_29603 (Lnet/minecraft/class_32$class_5143;Lnet/minecraft/class_3283;Lnet/minecraft/class_6904;Lcom/mojang/authlib/minecraft/MinecraftSessionService;Lcom/mojang/authlib/GameProfileRepository;Lnet/minecraft/class_3312;Ljava/lang/Thread;)Lnet/minecraft/class_1132;
 		ARG 1 thread
 		ARG 3 resourceManager
+		ARG 7 thread_
```

Comment:
What's the difference between ARG 7 and ARG 1?

Reply:
wait what there's ARG 1? Just noticed now. Cursed lambda param most likely, fixing